openedx/core/djangoapps/credentials_service/models.py

```diff
@@ -1,3 +1,264 @@
 """
 Models for the credentials service.
 """
+from __future__ import unicode_literals
+import uuid  # pylint: disable=unused-import
+
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.sites.models import Site
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from model_utils.models import TimeStampedModel
+from xmodule_django.models import CourseKeyField
```

Comment:
You cannot use this module. When this code is moved to an IDA, you will not have access to `xmodule_django`.